cmd/iam_apikey_create.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyCreateItemOutput egoscale.APIKey
+
+func (o *apiKeyCreateItemOutput) toJSON()  { outputJSON(o) }
+func (o *apiKeyCreateItemOutput) toText()  { outputText(o) }
+func (o *apiKeyCreateItemOutput) toTable() { outputTable(o) }
+
+// apiKeyCreateCmd represents an API key creation command
+var apiKeyCreateCmd = &cobra.Command{
+	Use:     "create <description>",
+	Short:   "Create an API key",
```

Comment:
You command is missing the `Long` field describing the available custom output fields. Again, please see the `affinitygroup create` command for a relevant example: https://github.com/exoscale/cli/blob/master/cmd/affinitygroup_show.go#L27-L30